src/core_plugins/kibana/public/dashboard/styles/index.less

```diff
@@ -31,6 +31,14 @@
   }
 }
 
+/**
+ * 1. For expanded panel mode, we want to hide the grid, but keep it rendered so it doesn't cause all visualizations
+ * to re-render once the panel is minimized, which is a time consuming operation.
+ */
+.layout-hidden {
```

Comment:
Maybe the naming should be less generic? `dashboard-layout-hidden` or something like that?